### PR TITLE
Removal of Ersilia current command

### PR DESCRIPTION
**Pull request by Ajoke23**
Thank you for taking your time to contribute to Ersilia, just a few checks before we proceed
- [x] Have you followed the guidelines in our [Contribution Guide](https://github.com/ersilia-os/ersilia/blob/master/CONTRIBUTING.md)
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

**Description**

Remove the ersilia current command

**Changes to be made**
- Delete current.cmd() line of code
- Delete all line of code in current.py
- Deleted the entire current.py file since it's not useful

**Status**
-Successfully removed the Ersilia current command
- I ran the `ersilia current` and got the following output:
successfully removal of current command

**To do**

Run it locally (Dnone)

Is this pull request related to any open issue? If yes, replace issueID below with the issue ID

Related to #1267

### CODE REVIEW

**Comment by Ajoke23** [comment]
Hi @DhanshreeA 
I have successfully removed the current command. I have attached output after testing it locally
successfully removal of current command

This is the main pull request. I have initially created a pull request yesterday for this same task. Can i go ahead of close the previous pull request #1282 or which pull request is preferable

CC @GemmaTuron

**Comment by Ajoke23** [comment]
Yes, it's. This is the pull request I want @DhanshreeA to got through.
I am waiting for feedback if I can go ahead and close pull request #1296

**Comment by Ajoke23** [comment]
> hi @Ajoke23 isn't this a duplicate PR to the one you previously opened?

Yes, it's. This is the pull request I want @DhanshreeA to got through.
I am waiting for feedback if I can go ahead and close pull request #1282

**Comment by adebisi4145** [comment]
Hello @Ajoke23, next time just rename the branch from github.

[Here's](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) a detailed explanation on how to go about it

**Review by DhanshreeA — APPROVED** [review]

**Review by dzumii — CHANGES_REQUESTED** [review]

**Comment by dzumii** [comment]
@Ajoke23 You can also run the `ersilia current` command (before and after you made changes) and post the output here

File: ersilia/cli/commands/current.py [hunk 1]
```diff
@@ -1,17 +1 @@
-from . import ersilia_cli
```
[the remark this answers is not in the document]
Comment (Ajoke23) [reply ersilia/cli/commands/current.py:1]: Hi @dzumii 
The entire file was removed

**Comment by Ajoke23** [comment]
> @Ajoke23 You can also run the `ersilia current` command (before and after you made changes) and post the output here

Hi @dzumii I posted the output after changes was made above

**Comment by dzumii** [comment]
@DhanshreeA I have tested and reviewed this PR and I think this is good to go

**Comment by DhanshreeA** [comment]
> Hi @DhanshreeA I have successfully removed the current command. I have attached output after testing it locally successfully removal of current command
> 
> This is the main pull request. I have initially created a pull request yesterday for this same task. Can i go ahead of close the previous pull request #1282 ?
> 
> CC @GemmaTuron

@Ajoke23 in the screen shot you have shared, it seems you are not running the `ersilia` command correctly itself. It is spelled as `erisilia` in your screen shot.

**Comment by Ajoke23** [comment]
Hi @DhanshreeA that was a typo error. I will try again and give you a feedback 
> > Hi @DhanshreeA I have successfully removed the current command. I have attached output after testing it locally successfully removal of current command
> > 
> > This is the main pull request. I have initially created a pull request yesterday for this same task. Can i go ahead of close the previous pull request #1282 ?
> > 
> > CC @GemmaTuron
> 
> @Ajoke23 in the screen shot you have shared, it seems you are not running the `ersilia` command correctly itself. It is spelled as `erisilia` in your screen shot.

**Comment by P-Mefut** [comment]
Okay great @Ajoke23, your work ready to be merged then... I've ran the test and it's in good standing...

**Comment by Ajoke23** [comment]
> Okay great @Ajoke23, your work ready to be merged then... I've ran the test and it's in good standing...

Thanks a lot. @dzumii also ran it too and I am good to go
@DhanshreeA will merge it soonest, I believe so.

**Pull request merged**